internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -425,18 +417,6 @@ func (r *AddOnDefinitionResource) Read(ctx context.Context, req resource.ReadReq
 	}
 	state.Labels = labels
 
-	if definition.TerraformCloudWorkspaceDetails != nil {
```

Comment:
This was accidentally included in a merged PR so I am removing it so it's back to how it used to function.

Reply:
Actually ditto here about not wanting to delete the whole block.